### DISCUSSION TITLE
Gerätetyp „Wärmepumpe“

### DISCUSSION THREAD

**Discussion by Exo84** [camueller/SmartApplianceEnabler#415] [Installation und Betrieb]
Sollte bei diesem Gerätetyp nicht auch "Energie" bei der Anforderungsart wählbar sein? Ich habe nur "Laufzeit".

Setup:

**Messen**

![image](https://user-images.githubusercontent.com/116544659/209858860-e12614f3-b119-4b36-afe3-8625c942af74.jpeg)

![image](https://user-images.githubusercontent.com/116544659/209858885-224114ee-ebe0-4ce0-a362-9f2413cb1ac9.jpeg)

![image](https://user-images.githubusercontent.com/116544659/209858920-02ff5424-e4c2-420a-aff8-0d20aa562978.jpeg)

![image](https://user-images.githubusercontent.com/116544659/209858942-e0f1010a-ce27-440f-9539-69b2a58764cf.jpeg)

**Sg Ready**

![image](https://user-images.githubusercontent.com/116544659/209859121-25c2c5a1-719d-404a-9b05-c7811b51dc3d.jpeg)

![image](https://user-images.githubusercontent.com/116544659/209859166-d4a063d9-af79-4469-91dd-67ac3c192af0.jpeg)

![image](https://user-images.githubusercontent.com/116544659/209859190-ef9b5cd8-39e1-450a-991b-421d300f2e33.jpeg)

![image](https://user-images.githubusercontent.com/116544659/209859209-46f39830-26a7-4fd7-afb0-00b27b7d019d.jpeg)

**Ergebnis**

![image](https://user-images.githubusercontent.com/116544659/209859284-4f7a07f0-47a4-4191-9229-96ecdc9ca72c.jpeg)

Logische sendet der SHM auch nie ein Einschaltsignal für den GIPO Schalter.

Wo liegt mein Fehler??

Vielen Dank!!

**Comment by camueller** [comment]
> Sollte bei diesem Gerätetyp nicht auch "Energie" bei der Anforderungsart wählbar sein? Ich habe nur "Laufzeit".

Nein, das ist so korrekt. Die Anforderungsarten Energie und SOC sind nur bei Verbrauchern mit variabler Leistungsaufnahme (z.B. Wallboxen) verfügbar.

**Reply by Exo84** [reply]
Ja, so habe ich es auch verstanden. 

Was wäre dann der richtige Typ für das Überschuss (SG ready) "Gerät"?

Aktuell habe ich "Wärmepumpe" gewählt (was es ja auch ist). Aber mit dem Typen kann ich ja nirgends sagen. Wenn, xx Überschuss anliegt dann schalte GIPO xy. Der SHM erkennt es demnach auch als WP und sagt nicht Steuerbar (siehe letztes Bild).

**Reply by camueller** [reply]
Der Typ hat keine Auswirkung auf den SAE ausser wenn er auf `EVCharger` gesetzt ist. Der Typ wir lediglich an den SHM übermittelt.

Wenn ein Gerät nur bei Überschuss eingeschaltet werden soll, muss das über den Zeitplan eingestellt werden: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Schedules_DE.md

**Reply by Exo84** [reply]
Siehe Screenshots. 

Dann sollte mein setup passen?

Das Gerät SG Ready ist im SHM gerade mit 100% PV eingestellt. Dann liegt es "nur" daran das er zu wenig PV Leistung bis dahin hatte? Kann ich irgendwie auslesen welche Leistung der SHM für das Gerät vorsehen/wünschen würde?

Zweitens:

Warum steht in der Sunny App dann das Gerät als nicht steuerbar? Wenn er es so deklariert wird er es doch wohl niemals ansteuern?

**Comment by camueller** [comment]
Auch bei komplexen Setups wie WP mit SG-Ready gelten die normalen Voraussetzungen, damit Du Einschaltbefehle vom SHM bekommen kannst: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md#semp3

Bei den aktuellen Witterungsbedingung hast Du vermutlich nicht genu Überschuss, um solche Einschaltbefehle vom SHM zu erhalten. Leider kann man auch nicht in den SHM schauen um herauszufinden, warum er keinen Einschaltbefehl sendet.

> Warum steht in der Sunny App dann das Gerät als nicht steuerbar? Wenn er es so deklariert wird er es doch wohl niemals ansteuern?

Ist ein Fehler in der SMA App.

**Reply by Exo84** [reply]
Okay, dann lass ich das setup mal so und warten auf schönes Wetter.

Guten Rutsch!
